app/views/questions/mailing_address/edit.html.erb

```diff
@@ -13,8 +13,6 @@
       <%= f.cfa_input_field(:zip_code, t("views.questions.mailing_address.zip_code"), classes: ["form-width--zip"]) %>
     </div>
 
-    <button class="button button--primary button--wide" type="submit">
-      <%=t("general.confirm") %>
-    </button>
+    <%= f.continue %>
```

Comment:
We remember talking to Anu about how the mailing address page can say Continue rather than Confirm.